### DISCUSSION TITLE
Windows Server Backup

### DISCUSSION THREAD

**Discussion by ttkkaakkuu** [dokan-dev/dokany#1106] [Q&A]
Hi!
I have a question about using dokany/mirror.exe on Windows Server Backup.
In Windows Server Backup, neither the backup destination nor the backup source shows the drive letter mounted by mirror.exe.
I don't understand the reason for this.

Environment:
 * Windows version: Windows Server 2019
 * Processor architecture:  x64
 * Dokany version: 2.05.1000
 * Library type (Dokany/FUSE): Dokany

Thank you.

**Comment by Liryna** [comment]
Hi @ttkkaakkuu ,
What command line are you using with mirror ?
Have you tried with the option `/o` ?

**Reply by ttkkaakkuu** [reply]
Hi @Liryna,
Thank you for your kindness!!
I had used the following command
  ` mirror.exe  /r D:\test /l M:`

And next, I  added `/o `.
But Windows Server Backup doesn't show the volume M: as a backup source or destination.

Currently, I am trying to find the cause.
If I get something about it let me ask again.
And if you have some advices , I would like to know.
